Update local installation guide - checking installation status + troubleshooting (#2047)
kyma-project/kyma: docs/kyma/docs/030-inst-local-installation-from-release.md

Follow the guidelines in the subsections to confirm that your Kubernetes API Server is up and running as expected.

### Verify the installation status using the is-installed.sh script

The `is-installed.sh` script is designed to give you clear information about the Kyma installation. Run it at any point to get the current installation status, or to find out whether the installation is successful.

> If the script indicates the installation failed, try installing Kyma again by re-running the `run.sh` script.

Reviewer: ```suggestion
If the script indicates that the installation failed, try to install Kyma again by re-running the `run.sh` script.
```

> 

Reviewer 2: Please also mention github issues

Author: it's done

> If the installation fails in a reproducible manner, don't hesitate to reach out on the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.

Reviewer 3: 👍

> If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out on the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.

Reviewer: ```suggestion
If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
```

To enable Horizontal Pod Autoscaler, follow these steps:

1. Enable the metrics server for resource metrics by running the following command:
  ```
  minikube addons enable metrics-server
  ```

2. Verify if the metrics server is active by checking the list of addons:
  ```
  minikube addons list
  ```

## Troubleshooting

If the Installer does not respond as expected, check the installation status using the `is-installed.sh` script with the `--verbose` flag added. Run:
```
scripts/is-installed.sh --verbose
```

If the installation is successful but a component does not behave in an expected way, see if all deployed Pods are running. Run this command:  
```
kubectl get pods --all-namespaces
```

The command retrieves all Pods from all Namespaces, the status of the Pods, and their instance numbers. Check if the STATUS column shows Running for all Pods. If any of the Pods that you require do not start successfully, perform the installation again.

> If the problem persists, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue or reach out on the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.

Reviewer: ```suggestion
If the problem persists, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
```
